lyonjs/shortvid.io
# feat(theme): create theming for the app

## 🤔 Why do you want to make those changes?

To give the user the choice to choose it's favourite theme.

## 🧑‍🔬 How did you make them?

I've used the library next-theme and created a button to switch between theme light and dark, then i updated the app variables for the colours to match the theme.

## 🧪 How to check them?

- Check the preview and see if everything is ok
- Check the code
